src/components/TextArea/TextArea.e2e.ts

```diff
@@ -0,0 +1,27 @@
+import { browser } from 'protractor';
```

Comment:
We need to think if this test is really needed, or maybe we can add cases to test in e2e.
Do we have any cases where we need real dom for the test?
If not, we can remove this file.

Reply:
Done